src/content/docs/en/reference/api-reference.mdx

```diff
@@ -619,6 +619,21 @@ This property is only available when building for SSR (server-side rendering) an
 
 The locale computed from the current URL, using the syntax specified in your `locales` configuration. If the URL does not contain a `/[locale]/` prefix, then the value will default to `i18n.defaultLocale`.
 
+### `Astro.routePattern`
+
+  <Since v="5.0.0" />
+
```

Comment:
```suggestion

<p>

**Type:** string <br />
  <Since v="5.0.0" />
  </p>

```
Is this correct, just trying to apply @ArmandPhilippot 's edit before he does?  :smile:

Reply:
😆 Looks good!